bin/play.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require_relative '../lib/tictactoe'
+require_relative '../lib/main_io'
+require_relative '../lib/board'
+require_relative '../lib/presenter'
+require_relative '../lib/game'
+require_relative '../lib/user_input'
+
+
+io = MainIO.new(input_stream: $stdin, output_stream: $stdout)
+
+game_setup = {
+  logic: TicTacToe.new,
+  user_input: UserInput.new(io: io),
+  presenter: Presenter.new(io: io),
+  board: Board.new(layout: [*1..9].map!(&:to_s))
+}
+
+Game.play(game_setup)
```

Comment:
This feels a bit weird. I would expect something like:
```Ruby
Game.new(game_setup).play
```

Reply:
I think I'm going to rename that as `tic_tac_toe` after the game logic